frontend/app/routes/_gcweb-app.personal-information.address.edit.tsx

```diff
@@ -1,33 +1,54 @@
-import { Form, Link } from '@remix-run/react';
+import { Form, Link, useLoaderData } from '@remix-run/react';
+
+import { LoaderFunctionArgs, json } from '@remix-run/node';
+
+import { getUserService } from '~/services/user-service.server';
+import { getEnv } from '~/utils/env.server';
+
+export async function loader({ request }: LoaderFunctionArgs) {
+  const env = getEnv();
+  const userService = getUserService({ env });
+  const userId = await userService.getUserId();
+  return json({
+    user: await userService.getUserInfo(userId),
+  });
+}
 
 export default function ChangeAddress() {
+  const { user } = useLoaderData<typeof loader>();
+
+  const defaultValue = {
```

Comment:
**Nitpick:** (since you have more than one value in the object)

```suggestion
  const defaultValues = {
```

Reply:
update in commit https://github.com/DTS-STN/canadian-dental-care-plan/pull/102/commits/51883b72fe4a8a91e59dd656e6ee040b03ae9346